libpod/container.go

```diff
@@ -392,8 +392,54 @@ func (c *Container) teardownStorage() error {
 	return nil
 }
 
+// Refresh refreshes the container's state after a restart
+func (c *Container) refresh() error {
+	c.lock.Lock()
+	defer c.lock.Unlock()
+
+	if !c.valid {
+		return errors.Wrapf(ErrCtrRemoved, "container %s has been removed from the state", c.ID())
```

Comment:
I'm confused by this message.  What state was it removed from?  Should this just be "container %s is not found" or something to that effect?